metacat-common/src/main/java/com/netflix/metacat/common/dto/ResolveByUriResponseDto.java

```diff
@@ -0,0 +1,21 @@
+package com.netflix.metacat.common.dto;
+
+import com.netflix.metacat.common.QualifiedName;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.List;
+
+/**
+ * ResolveByUriResponseDto.
+ *
+ * @author zhenl
+ * @since 1.0.0
+ */
+@Data
```

Comment:
if this is a DTO please make the variables final. That way lombok won't create setters.